src/TMediaData.h

```diff
@@ -40,37 +40,45 @@ class TMediaData
 
     enum MediaContinue { MediaContinueDefault = true, MediaContinueRestart = false };
 
+    enum MediaFade { MediaFadeMin = 0, MediaFadeNotSet = 0 };
```

Comment:
That looks a bit dodgy to me, is this an `int` or an `enum MediaFade` - the whole (well, maybe a significant part) of the point of using `enum`s is to confine a variable to a precise *enumerated* number of states whose values are usually kept opaque so that coders only work with those states. In some cases the constant values are assigned to integral powers of 2 so that bitwise operations can be done on combinations (and which lend themselves to Qt's own `QFlag`/`QFlags` classes).

[Wikipedia](https://en.wikipedia.org/wiki/Enumerated_type#C++) has some information about this. Also, https://en.cppreference.com/w/cpp/language/enum suggests - as far as I can see - that given these two, identical values, an `enum MediaFade` may not be allowed to store anything other than the value `0`! :astonished:

Reply:
Made this change, thank you.